morpheushelper/cogs/mod.py

```diff
@@ -260,11 +260,11 @@ async def report(self, ctx: Context, member: Member, *, reason: str):
     @commands.command()
     @Permission.warn.check
     @guild_only()
-    async def warn(self, ctx: Context, member: Member, *, reason: str):
+    async def warn(self, ctx: Context, member: Union[Member, int], *, reason: str):
         """
         warn a member
         """
-
+        member: Member = await self.get_user(ctx.guild, member)
```

Comment:
Maybe another var name like user makes more sense because it doesn't overload the method param.
Also the type is a `Member` or a `User` so the typing should be like this:
```python
user: Union[Member, User] = await self.get_user(ctx.guild, member)
```